x/concentrated-liquidity/tick.go

```diff
@@ -253,6 +253,68 @@ func (k Keeper) GetTickLiquidityForRange(ctx sdk.Context, poolId uint64) ([]quer
 	return liquidityDepthsForRange, nil
 }
 
+// GetLiquidityNetInDirection returns an array of liquidity depth in the given zeroForOne direction.
+// Uses boundTick if given, and iterates until we hit bound Tick's index. If not, uses min tick || max tick depending on the direction of zeroForOne.
+func (k Keeper) GetLiquidityNetInDirection(ctx sdk.Context, poolId uint64, zeroForOne bool, boundTickPointer *sdk.Int) ([]query.LiquidityDepth, error) {
```

Comment:
@jonator question for you.

What would be the preferred way for you to specify the direction of the swap?

The direction of the swap is determined by which token is being swapped in. Here, we are assuming that FE will provide a boolean with the knowledge of the direction.

However, maybe we don't want frontends to care about this and, instead, just provide the denom of the token being swapped in. WDYT?

Reply:
I think denom of token swapped in could be a bit more robust for FE use. Passing in a boolean assumes the frontend has prior knowledge of the assets' in the pool being positioned as the 0 or 1 asset.

Reply:
Changed this so that it now takes in token in denom instead of taking in ZeroForOne